example/image-classification/symbols/pvanet.py

```diff
@@ -0,0 +1,140 @@
+import mxnet as mx
+def get_symbol(num_classes, **kwargs):
+
+    data = mx.sym.Variable(name='data')

```

Comment:
why put the same file at two places?

Reply:
pvanet is a classification net, it is a little different with it's counterpart as in symbol_pvanet, And secondly, I cannot upload pre-trained model(pvanet.py) for it is too large, thus some one else could pretrain with pvanet.py on ImageNet